terraform/azure/modules/mysql-db/main.tf

```diff
@@ -0,0 +1,76 @@
+
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "azrg" {
+  name     = var.resource_group_name
+  location = var.location
+}
+
+resource "azurerm_virtual_network" "vnet" {
```

Comment:
please dont create vnets here, use the existing modules under modules/vnet .

In fact try to reuse as much as possible. If your missing a module create that first.

Reply:
used existing vnet module